code/go/0chain.net/core/ememorystore/connection.go

```diff
@@ -4,9 +4,10 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/0chain/gorocksdb"
+
 	"0chain.net/core/common"
 	"0chain.net/core/datastore"
-	"github.com/0chain/gorocksdb"
```

Comment:
Does not seem like a meaningful change. Suggest reverting this.

Reply:
I've sorted imports because it increases readability.
The first group for core go libs, the second for external imports, and the third for internal imports.
I can revert this, but I think sorting is good.

Reply:
I completely agree with Benjamin - strongly recommended sort imports automatically by `goimports`

Reply:
Doing unrelated refactoring in files you are not working on is not recommended. 

No problem with `goimports`, I recommend using it systematically, maybe raise an issue about it.

Reply:
Doing unrelated refactoring in files you are not working on is not to be recommended. 

No problem with `goimports`. Maybe we can add it to CI.